custom_components/flashforge_adventurer_3/const.py

```diff
@@ -1,3 +1,14 @@
-DOMAIN = 'flashforge_adventurer_3'
+# const.py
+
+DOMAIN = 'FlashForge_adventurer_3'
 CONF_PRINTERS = '3dprinters'
-DEFAULT_PORT = 8899
\ No newline at end of file
+DEFAULT_PORT = 8899
+
+# Add constants for printer models
+CONF_MODEL = 'model'
+MODEL_FINDER = 'FlashForge Finder'
+MODEL_ADVENTURER_3 = 'FlashForge Adventurer 3'
+MODEL_ADVENTURER_4 = 'FlashForge Adventurer 4'
+MODEL_ADVENTURER_5M = 'FlashForge Adventurer 5M'
+MODEL_ADVENTURER_5M_PRO = 'FlashForge Adventurer 5M Pro'
+AVAILABLE_MODELS = [MODEL_FINDER, MODEL_ADVENTURER_3, MODEL_ADVENTURER_4, MODEL_ADVENTURER_5M, MODEL_ADVENTURER_5M_PRO]
```

Comment:
At the moment the human-readable name of the printer model is the same as its identifier used in the code, for example to choose the most appropriate regexp. Let's separate these two concerns.

Firstly, let's use shorter identifiers throughout the codebase, as well as an identifier persisted in the integration:

- `finder`
- `adventurer_3`
- `adventurer_4`
- `adventurer_5m`
- `adventurer_5m_pro

Up to you if you want to export these as constants and use those constants, or if you want to use string literals in `protocol.py`.

Secondly, let's create a map of those identifiers to human-readable printer names that will be used in the UI, with values being the identifiers above and labels being the strings that you have now.